Mathlib/Logic/Small/Basic.lean

```diff
@@ -83,6 +83,9 @@ instance (priority := 100) small_max (α : Type v) : Small.{max w v} α :=
   small_lift.{v, w} α
```

Comment:
This instance seems to be broken, since if you put exactly the same type below and ask Lean to infer it, Lean can't do it:
```
instance (priority := 100) small_max' (α : Type v) : Small.{max w v} α := inferInstance -- fails
```